### DISCUSSION TITLE
Settings, extensions etc. sync

### DISCUSSION THREAD

**Discussion by jborbik** [VSCodium/vscodium#793] [Q&A] [answered]
Is there any way to sync the settings, extensions etc. with vscodium? Does copying ~/.vscode-oss work, even cross platform (Ubuntu & MacOS)?

**Comment by daiyam** [comment, marked as answer]
You have [Sync Settings](https://marketplace.visualstudio.com/items?itemName=zokugun.sync-settings) made by me. It's new, it's lacking some features. But it get the job done.
